_posts/en/newsletters/2024-07-19-newsletter.md

```diff
@@ -159,10 +159,10 @@ repo], and [BINANAs][binana repo]._
   [trampoline routing][topic trampoline payments]. {% assign timestamp="1:11:58" %}
 
 - [Rust Bitcoin #2646][] introduces new inspectors for script and witness
-  structures such as `redeem_script` to ensure that compliance with [BIP16][]
-  rules regarding P2SH spending, `taproot_control_block` and `taproot_annex` to
-  ensure compliance with [BIP341][] rules and `witness_script` to ensure that
-  the P2WSH witness script complies with [BIP141][] rules. See Newsletter
+  structures such as `redeem_script` to ensure compliance with [BIP16][];
+  rules regarding P2SH spending, `taproot_control_block`, and `taproot_annex` to
+  ensure compliance with [BIP341][] rules; and `witness_script` to ensure
+  P2WSH witness scripts comply with [BIP141][] rules. See Newsletter
```

Comment:
Reviewers, carefully check this part of the change that makes "the witness script" plural, that is, "witness scripts" -- I think that's the intended meaning, but please check.

I think the other part of this change is correct; there's a list within a list, so semicolons should be used for the top-level list. (It took me several re-readings to parse this in what I think is the correct way.)